src/core/qgsexpressioncontext.h

```diff
@@ -378,6 +378,11 @@ class CORE_EXPORT QgsExpressionContext
      */
     void appendScope( QgsExpressionContextScope* scope );
 
+    /**
+     * Remove the last scope from the expression context.
+     */
+    void popScope();
```

Comment:
Can you add a unit test for this? QgsExpressionContext has good test coverage, I don't want this to drop.